forum/templates/forum/post_detail.html

```diff
@@ -0,0 +1,53 @@
+{% extends 'layout.html' %}
+{% load humanize %}
+
+{% block meta_title %}{{ object.title }}{% endblock %}
+{% block meta_description %}Post description{% endblock %}
+{% block meta_og_title %}{{ object.title }}{% endblock %}
+{% block meta_og_description %}Post description{% endblock %}
+
+{% block extra_style %}
+    <style>
+        div.comment-item {
+            border: 1px solid #000000;
+            padding: 0em 2em;
+        }
+
+        p.comment-body {
+
+        }
+
+        p.comment-footer {
+            font-size: 0.8em;
+        }
+    </style>
+{% endblock %}
+
+{% block body %}
+    <section>
+        <h1>{{ object.title }}</h1>
+        <p>Post description</p>
+    </section>
+
+    <section>
+        {% for comment in object.comment_set.all %}
+            <div class="comment-item">
+                {% if user.is_authenticated %}
```

Comment:
We don't want to show the upvote+downvote buttons unless the User is logged in.

Reply:
I thought there was an user.is_active...but it looks like authenticated is correct.  I see you also added csrf for each item.  did not know it would make sense to create a different url for voting using comment.post id.